Redundant use of argument optimistic_loop and setting assumeUnwindCond
Redundant use of argument optimistic_hashing and setting optimisticUnboundedHashing with value false
Saving last configuration file to .certora_internal/23_06_07_19_01_13_711/.last_confs/last_conf_07_06_2023__19_01_13.conf
contract name StaticATokenLMHarness is the same as the file name and can be omitted from certora/harness/StaticATokenLMHarness.sol:StaticATokenLMHarness
contract name RewardsControllerHarness is the same as the file name and can be omitted from certora/harness/rewards/RewardsControllerHarness.sol:RewardsControllerHarness
contract name TransferStrategyHarness is the same as the file name and can be omitted from certora/harness/rewards/TransferStrategyHarness.sol:TransferStrategyHarness
contract name AToken is the same as the file name and can be omitted from lib/aave-v3-core/contracts/protocol/tokenization/AToken.sol:AToken
contract name SymbolicLendingPool is the same as the file name and can be omitted from certora/harness/pool/SymbolicLendingPool.sol:SymbolicLendingPool
contract name DummyERC20_rewardToken is the same as the file name and can be omitted from certora/harness/tokens/DummyERC20_rewardToken.sol:DummyERC20_rewardToken
contract name DummyERC20_aTokenUnderlying is the same as the file name and can be omitted from certora/harness/tokens/DummyERC20_aTokenUnderlying.sol:DummyERC20_aTokenUnderlying
There is no TAC file. Going to script EVMVerifier/certoraBuild.py to main_with_args()
Creating dir /home/dev1644/workspace/workspace-test/aave-certora/.certora_internal/23_06_07_19_01_13_711/.certora_config
In /home/dev1644/workspace/workspace-test/aave-certora/certora/specs/erc4626/erc4626_verified.spec, found the imports: [('erc4626_base.spec', '1:1')]
In /home/dev1644/workspace/workspace-test/aave-certora/certora/specs/erc4626/erc4626_base.spec, found the imports: [('../methods/methods_base.spec', '1:1')]
In /home/dev1644/workspace/workspace-test/aave-certora/certora/specs/methods/methods_base.spec, found the imports: [('erc20.spec', '1:1')]
In /home/dev1644/workspace/workspace-test/aave-certora/certora/specs/methods/erc20.spec, found the imports: []
copying spec file /home/dev1644/workspace/workspace-test/aave-certora/certora/specs/erc4626/erc4626_verified.spec to /home/dev1644/workspace/workspace-test/aave-certora/.certora_internal/23_06_07_19_01_13_711/.certora_config/0_erc4626_verified.spec.spec
writing /home/dev1644/workspace/workspace-test/aave-certora/.certora_internal/23_06_07_19_01_13_711/.certora_verify.json
Path to typechecker is /home/dev1644/.local/lib/python3.10/site-packages/certora_jars/Typechecker.jar
running ['java', '-jar', '/home/dev1644/.local/lib/python3.10/site-packages/certora_jars/Typechecker.jar', '-v', '.certora_internal/23_06_07_19_01_13_711/.certora_verify.json', '-m', '.certora_internal/23_06_07_19_01_13_711/.certora_metadata.json', '-d', '.certora_internal/23_06_07_19_01_13_711']
Failed to compile spec file
[main] ERROR ALWAYS - certora/specs/erc4626/erc4626_verified.spec:49:1: Syntax error: unexpected token near )
[main] ERROR ALWAYS - certora/specs/erc4626/erc4626_verified.spec:49:1: Couldn't repair and continue parse unexpected token near )
[main] ERROR ALWAYS - CVL parser failed with exception. Exception message: "Failed to parse certora/specs/erc4626/erc4626_verified.spec. Exiting."
spec.CVLAstBuilder$CVLParsingException: java.lang.Exception: Failed to parse certora/specs/erc4626/erc4626_verified.spec. Exiting.
build failed
Failure traceback: 
Shared.certoraUtils.CertoraUserInputError: Syntax error in specification file
Encountered an error running Certora Prover:
Syntax error in specification file
